src/components/project/controls/AttachmentPreviewButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {Button} from '../../controls/form-controls/Button';
+import {Icon} from '../../controls/Icon';
+import {t} from '../../utils';
+
+
+type AttachmentPreviewButtonProps = {
+  downloadUrl: string;
+  /** Specifies translation key for the tooltip  */
+  toolTipTransString: string;
```

Comment:
`tooltip` is hier goed genoeg?
We zien aan het type dat het een "string" is. (wat type zou een "tooltip" anders kunnen zijn?)

Reply:
Alright klopt